installing/installing_azure/installing-restricted-networks-azure-user-provisioned.adoc

```diff
@@ -0,0 +1,152 @@
+:_content-type: ASSEMBLY
+[id="installing-restricted-networks-azure-user-provisioned"]
+= Installing a cluster on Azure in a restricted network with user-provisioned infrastructure
+include::_attributes/common-attributes.adoc[]
+:context: installing-restricted-networks-azure-user-provisioned
+
+toc::[]
+
+In {product-title} {product-version}, you can install a cluster on Microsoft Azure that uses infrastructure that you provide and an internal mirror of the installation release content.
+
+[IMPORTANT]
+====
+You can install an {product-title} cluster by using mirrored installation release content, but your cluster requires internet access to use the Azure APIs.
+====
+
+Several link:https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/overview[Azure Resource Manager] (ARM) templates are provided to assist in completing these steps or to help model your own. You can create the required resources through other methods.
+
+[IMPORTANT]
+====
+The steps for performing a user-provisioned infrastructure installation are provided as an example. Installing a cluster with infrastructure you provide requires knowledge of the cloud provider and the installation process of {product-title}.
+====
```

Comment:
I had used the existing docs for user-provisioned installation. You can follow the same from here: https://docs.openshift.com/container-platform/4.13/installing/installing_azure/installing-azure-user-infra.html